CONTRIBUTING.md

```diff
@@ -206,44 +209,43 @@ For smaller contributions, you may take shortcuts.
 The workflow is explained in detail below. In summary, it consists of these steps.
 
 1. Create a branch
-2. Create a draft pull request
+2. Create a "WIP" (work in progress) merge request. This means that the MR title starts with "WIP:", and is treated specially by gitlab (for instance, by preventing it from getting merged while in "WIP" state)
 3. Make the changes, commit with amend and rebase
 4. Push regularly
-5. Clean up the commits, push, and set pull request to "Ready for review"
+5. Clean up the commits, push, and move the MR out of "WIP" state
 6. Review starts - reviewer "Requests changes"
 7. Author addresses review comments in additional fixup commits.
    * If no more fixes are needed
-     * Author rebases with `git rebase -i --autosquash master` to clean up the pull request.
+     * Author rebases with `git rebase -i --autosquash master` to clean up the merge request.
    * If more fixes are needed (suggested by reviewers or by the author himself)
      * Author does rebase-force-push to squash fixup commits and asks for a followup review (this makes the next review iterations simpler)
```

Comment:
I know this is how we originally stated the flow to be - but I am not sure the fixup-commits after the review are really used (I didn't encounter them if the review process for a long time now.
Usually, the flow I see is:
review comments are addressed in one of the 3 ways:
1. decision not to accept the comment (and no change is done according to that comment)
2. accept the comment requested change, and if it is a simple fix - resolve the comment in the conversation (but fix is already squashed to the commit that introduced the change). If the fix is not simple the conversation isn't resolved.
3. comment required more discussion and then a ping-pong is started until either '1' or '2' above is decided.

either way didn't see a PR that the changes were introduced as fixup, If this is the flow we want to use - we need to enforce it for future PR/merge requests.